mobile/mobile.go

```diff
@@ -144,3 +147,69 @@ func (w *Wrapper) PairDesktop(pkb64 string) (string, error) {
 
 	return topic, nil
 }
+
+// TODO: doesn't use a cleaned version of the phone number, if pn is supplied
+func (w *Wrapper) CheckPassword(password string, identity string) (bool, error) {
+	match := zxcvbn.PasswordStrength(password, []string{identity})
+	if match.Score < 3 {
+		return false, errors.New(fmt.Sprintf("weak password - crackable in %s", match.CrackTimeDisplay))
+	}
+	return true, nil
+}
+
+func (w *Wrapper) SignUp(username string, password string, identity string, identityType string) (int, *models.Response, error) {
```

Comment:
could make this `SignUpWithEmail` to make this simpler on the RN side, then add separate phone method if we ever do that